apps/src/code-studio/pd/workshop_dashboard/workshop_filter.jsx

```diff
@@ -79,6 +81,9 @@ export class WorkshopFilter extends React.Component {
     if (this.props.permission.has(WorkshopAdmin)) {
       this.loadOrganizers();
     }
+    if (this.props.permission.hasAny(WorkshopAdmin)) {
+      this.loadFacilitators();
```

Comment:
Why not include this in the conditional on line 81? we are checking the same condition.